### PR TITLE
Temporary object store credentials endpoint

### PR DESCRIPTION
**Pull request by hardbyte**
This PR adds an endpoint to retrieve temporary object store credentials.

The design is documented [here](https://github.com/data61/anonlink-entity-service/blob/develop/docs/designs/anonlink-upload-data-plan.md#addition-of-an-endpoint-to-retrieve-temporary-object-store-credentials). The main thrust of this is complete, but I still haven't finished the k8s deployment side of things. In particular I'd like to get the k8s initialization working and remove any object store credentials before adding support to open the minio object store to the internet.

### CODE REVIEW

File: backend/entityservice/views/objectstore.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          "Sid": "Upload-access-to-specific-bucket-only"
+        }
+      ]
+    }
+
+    return json.dumps(restricted_upload_policy)
+
+
+def authorize_external_upload(project_id):
+    assert config.UPLOAD_OBJECT_STORE_ENABLED, "Feature disabled"
```
Comment (wilko77) [inline backend/entityservice/views/objectstore.py:37]: will that create an appropriate response for the request?

File: tools/ci.yml [hunk 2]
```diff
@@ -16,14 +15,19 @@ services:
     image: data61/anonlink-app:${TAG:-latest}
     environment:
       - SERVER=http://nginx:8851
-      - INITIAL_DELAY=20
```
Comment (wilko77) [inline tools/ci.yml:19]: you removed the env var `INITIAL_DELAY` in a couple of places. Who uses this? What is it defaulting to now?

File: backend/entityservice/api_def/openapi.yaml [hunk 3]
```diff
@@ -1076,3 +1104,32 @@ components:
...
+          description: Object Store credentials (compatible with both AWS & MinIO)
+          type: object
+          properties:
+            AccessKeyId:
+              type: string
+            SecretAccessKey:
+              type: string
+            Expiration:
+              type: string
+            SessionToken:
```
[the remark this answers is not in the document]
Comment (hardbyte) [reply backend/entityservice/api_def/openapi.yaml:1134]: Exactly the same - thankfully

File: backend/entityservice/views/objectstore.py (hunk 1 above)
Comment (hardbyte) [reply backend/entityservice/views/objectstore.py:37]: I checked that the endpoint returns a `500` and the schema is correctly a `Problem`. However it doesn't actually send the "feature disabled" message to the user. I'll look into it.

File: tools/ci.yml (hunk 2 above)
Comment (hardbyte) [reply tools/ci.yml:19]: It is an optional environment variable used by both the integration tests and end to end tests - I probably shouldn't have deleted them as they serve as documentation.

Note I also added a small sleep in the docker `command` for the integrationtests container

File: backend/entityservice/views/objectstore.py (hunk 1 above)
Comment (hardbyte) [reply backend/entityservice/views/objectstore.py]: I forgot we had a `send_fail_request` for this exact purpose - fixed now.

**Pull request merged**